router/batchrouter/asyncdestinationmanager/klaviyobulkupload/klaviyobulkupload.go

```diff
@@ -106,7 +106,9 @@ func (kbu *KlaviyoBulkUploader) Poll(pollInput common.AsyncPoll) common.PollStat
 	importStatuses := make(map[string]string)
 	failedImports := make([]string, 0)
 	for _, importId := range importIds {
-		importStatuses[importId] = "queued"
+		if importId != "" {
```

Comment:
does this mean all the events that were part of unsuccessful upload gets re-tried in next attempt as they don't get marked as queued ?

Reply:
yes, they will be retried